src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/BehaviourServiceTest.kt

```diff
@@ -16,40 +16,49 @@ import java.time.ZoneId
 
 class BehaviourServiceTest {
   private val prisonApiService: PrisonApiService = mock()
-  private var clock: Clock = Clock.fixed(Instant.parse("2022-08-01T12:45:00.00Z"), ZoneId.systemDefault())
+  private var clock: Clock = Clock.fixed(Instant.parse("2022-08-01T12:45:00.00Z"), ZoneId.of("Europe/London"))
```

Comment:
better to use a fixed time zone as differences between dates could vary otherwise